### DISCUSSION TITLE
Changing the name of the binary in NFPM build

### DISCUSSION THREAD

**Discussion by uded** [goreleaser/goreleaser#4355] [Q&A] [answered]
Currently, as noted in the `contents` section in [NFPM customization guide](https://goreleaser.com/customization/nfpm/) binaries will be automatically included in the package. Using a simple template, I can also include it with my own name with the path I need.

Can I change the binary name included with NFPM **or** can I tell GoReleaser to stop including binary packages?

Rationale - due to some name conventions, I need to have a different name for the binary for Debian. Long story short - it's not the same and never will be as the project's name.

**Comment by caarlos0** [comment, marked as answer]
this is something we need to work on for v2... for now a workaround would be setting the package to `meta` and adding the binaries manually
